_data/simple-icons.json

```diff
@@ -6155,6 +6155,11 @@
             "hex": "D0072A",
             "source": "https://themighty.com/"
         },
+        {
+            "title": "The Models Resource",
+            "hex": "3971BE",
```

Comment:
If I do some eyedropping I get `#3A75BD`.... 🤔

Reply:
Ok, I fixed the color to what you suggested.